plane/source/docs/guide-tailsitter.rst

```diff
@@ -50,7 +53,52 @@ QHOVER and QLOITER.
 .. youtube:: bMsfjwUAfkM
     :width: 100%
 
-However, it can also have copter-like motors, like a conventional QuadPlane if :ref:`Q_TAILSIT_MOTMX<Q_TAILSIT_MOTMX>` is non-zero. Then this parameter determines which motors remain active in normal forward flight (plane modes). If non-zero, then use the :ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>` parameter to configure the multicopter motor style, and the appropriate MOTORx outputs will be activated.
+
+However, it can also have copter-like motors, like a conventional QuadPlane if :ref:`Q_TAILSIT_MOTMX<Q_TAILSIT_MOTMX>` is non-zero. Then this parameter determines which motors remain active in normal forward flight (plane modes). If non-zero, then use the :ref:`Q_FRAME_CLASS<Q_FRAME_CLASS>` and :ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>` parameter to configure the multicopter motor style, and the appropriate MOTORx outputs will be activated.
+
+.. note:: Currently, only the following combinations are supported for CopterMotor tailsitters:
+
++------------------------+------------------------------------+----------------------------------+
+| Motors                 |:ref:`Q_FRAME_CLASS<Q_FRAME_CLASS>` |:ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>` |
++------------------------+------------------------------------+----------------------------------+
+|  Quad                  | 1                                  | 0, 1                             |
++------------------------+------------------------------------+----------------------------------+
+|  Tri                   | 7                                  | ignored                          |
++------------------------+------------------------------------+----------------------------------+
+
+Motor Rotation and Layout
+=========================
+
+(looking down on nose from above)
+
+Single/Dual Motor Tailsitter
+----------------------------
+
+Motors are controlled by the Throttle, Throttle Left, Throttle Right outputs:
+
+.. image:: ../images/tailsit-motors.jpg
+
+
+QUAD I Motor Tailsitter
+-----------------------
+Motors are controlled by the M1-M4 outputs:
+
+.. image:: ../images/i-copter-quadplane.jpg
+
+
+QUAD X Motor Tailsitter
+-----------------------
+Motors are controlled by the M1-M4 outputs:
```

Comment:
the only example of this that I know of is @losawing's bi-wing which has motors 1 and 3 on the lower wing and 2,4 on the top wing. Also, he is using a PR branch which does use differential torque for yaw and hence the standard copter motor layout and rotation directions.

Reply:
but this is correct as far as you know for 4.0, right?

Reply:
yes, just untested